contracts/root/depositManager/DepositManager.sol

```diff
@@ -16,12 +16,13 @@ import {IDepositManager} from "./IDepositManager.sol";
 import {DepositManagerStorage} from "./DepositManagerStorage.sol";
 import {StateSender} from "../stateSyncer/StateSender.sol";
 import {Lockable} from "../../common/mixin/Lockable.sol";
-
+import {Drainable} from "../../common/misc/Drainable.sol";
 contract DepositManager is
     DepositManagerStorage,
     IDepositManager,
     IERC721Receiver,
-    ContractReceiver
+    ContractReceiver,
+    Drainable
```

Comment:
The entire point of this exercise was to replace the implementation of depositManager and then drain. Please don't inherit from `Drainable`. - In the test deploy a new instance of drainable, update address in depositManagerProxy and then call drain functions :)